### PR TITLE
Personalization of Tokens and RelationshipTemplates

### PR DESCRIPTION
**Pull request by Magnus-Kuhn**
While the creators of the Tokens and Templates can also load the Token/Template, I didn't mention this to avoid bloat. I hope that owners won't know/ won't be scared by the fact that they will need to get them from the backbone if they onboard a new device

### CODE REVIEW

**Comment by jkoenig134** [comment]
> I hope that owners won't know/ won't be scared by the fact that they will need to get them from the backbone if they onboard a new device

yeah, this is no problem. App users don't care and Connectors doesn't sync.

**Review by jkoenig134 — CHANGES_REQUESTED** [review]
Only two small details. Additionally I have the feeling that there are some pages missing (createTokenQRcodeFor*)?

File: _docs_integrate/establish-relationships.md [hunk 1]
```diff
@@ -81,6 +81,20 @@ If you have successfully created the [RelationshipTemplate]({% link _docs_integr
 
 {% include copy-notice description="Save the `id` of the RelationshipTemplate so that you can refer to it and make it available to other Identities later. For the same reason, save the value of the property `truncatedReference`." %}
 
+### Personalize the RelationshipTemplate
```
Comment (britsta) [inline _docs_integrate/establish-relationships.md:84]: Please put this section before the `### Successfully Created RelationshipTemplate` section and after the `### Input for Creating a RelationshipTemplate` section.

**Review by britsta — CHANGES_REQUESTED** [review]

File: _docs_use-cases/use-case-transport-create-token-qr-code-for-own-relationshiptemplate.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 # Start automatic generation
 permalink: use-case-transport-create-token-qr-code-for-own-relationshiptemplate
-published: false
+published: true
```
[the remark this answers is not in the document]
Comment (Magnus-Kuhn) [reply _docs_use-cases/use-case-transport-create-token-qr-code-for-own-relationshiptemplate.md:4]: The excel has been updated o7

**Comment by Magnus-Kuhn** [comment]
> Additionally I have the feeling that there are some pages missing (createTokenQRcodeFor*)?

There are only Template and File for * and both are here

**Review by britsta — APPROVED** [review]

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**